src/main/java/com/jadventure/game/conversation/ConversationManager.java

```diff
@@ -110,26 +110,10 @@ private Line getLine(int index, JsonArray conversation) {
     }
 
     public void startConversation(NPC npc, Player player) throws DeathException {
-        List<Line> conversation = null;
-        //Workaround as <code>lines.get(npc)</code> is not working.
-        Iterator it = lines.entrySet().iterator();
-        while (it.hasNext()) {
-            @SuppressWarnings("unchecked")
-            Map.Entry<NPC, List<Line>> entry = (Map.Entry<NPC, List<Line>>) it.next();
-            if (entry.getKey().equals(npc)) {
-                conversation = entry.getValue();
-            }
-            it.remove();
-        }
+        List<Line> conversation = getLines(npc);
         if (conversation != null) {
             Line start = null;
```

Comment:
Why not combine this like the refactoring above?
```suggestion
            Line start = getLine(npc, player, conversation);
```

Reply:
I've removed the start parameter here, see me suggestion on your getLine method.